src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -69,7 +69,9 @@ public function copy($originFile, $targetFile, $overwriteNewerFiles = false)
             }
 
             // Like `cp`, preserve executable permission bits
-            @chmod($targetFile, fileperms($targetFile) | (fileperms($originFile) & 0111));
+            if (stream_is_local($originFile)) {
```

Comment:
TL;DR @javiereguiluz's point: stream_is_local return false for `file://` paths, but chmod works on them. So yes, we should have a case for them.